async-inference/Async-Inference-Walkthrough-SageMaker-Python-SDK.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<p>Typically we use a demo S3 prefix formatted like:  <code>DEMO-async-inference</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>